src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift

```diff
@@ -54,22 +54,22 @@ class PresentationDetailController : UIViewController{
     let Grade: UITextView! = {
         let gf = UITextView()
         gf.textColor = UIColor.black
-        gf.text = "Grade: 3"
```

Comment:
not really clear what gf is... maybe use a more descriptive variable name? using context clues i can figure it out but just a thought

Reply:
gf is a UITextView that Grade gets set to at the end of that block. It's kind of unnecessary to create a new UITextView inside that variable so I replaced all names like that with the $0 macro, which is a common Swift macro